src/storage/rdb.cc

```diff
@@ -173,9 +173,13 @@ StatusOr<std::string> RDB::loadEncodedString() {
   }
 
   // Normal string
-  std::vector<char> vec(len);
-  GET_OR_RET(stream_->Read(vec.data(), len));
-  return std::string(vec.data(), len);
+  if (len == 0) {
+    return "";
+  }
+  std::string read_string;
+  read_string.resize(len);
+  GET_OR_RET(stream_->Read(read_string.data(), len));
```

Comment:
I wonder if it's a RdbStringStream, it might call `memcpy` on `std::vector<char>`, and when size is `0`, maybe memcpy will `memcpy` 0 in a invalid address, which might be a ub?